src/udiskslinuxfilesystemhelpers.c

```diff
@@ -128,6 +128,8 @@ take_filesystem_ownership (const gchar  *device,
   gboolean unmount = FALSE;
   gboolean success = TRUE;
 
+#define _FS_OWNERSHIP_MOUNT_OPTS "nodev,nosuid,noexec"
```

Comment:
This looks a bit unnecessary, but I guess it helps with future changes and highlights the specific "secure" options.